core/src/main/scripts/importer/validateData.py

```diff
@@ -2094,7 +2094,7 @@ def parseFeatureColumns(self, nonsample_col_vals):
         # products if the antibody name has a particular format.
         value = nonsample_col_vals[0].strip()
         if '|' not in value:
-            self.logger.error('No pipe symbol in RPPA probe column',
+            self.logger.error('No pipe symbol in Composite.Element.REF probe column',
```

Comment:
maybe the word "probe" is strange in the context of mass spec...just remove the word probe in this sentence. I was also wondering whether it makes sense to have `Composite.Element.REF` as column name in case of mass spec. Is there any probe information in `Composite.Element.REF` column for CPTAC data?